use-cases/vehicle-routing/src/main/java/org/acme/vehiclerouting/rest/VehicleRoutePlanResource.java

```diff
@@ -100,6 +96,50 @@ public String solve(VehicleRoutePlan problem) {
         return jobId;
     }
 
+    @Operation(summary = "Request recommendations to the RecommendedFit API for a new customer.")
+    @APIResponses(value = {
+            @APIResponse(responseCode = "200",
+                    description = "The list of fits for the given customer.",
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON,
+                            schema = @Schema(implementation = List.class)))})
+    @POST
+    @Consumes({MediaType.APPLICATION_JSON})
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("recommendation")
```

Comment:
I'd use `GET` here, since you're asking for data.

Reply:
We should not use the request body in the' GET' actions. That's the reason I chose the `POST` instead.